graviti/dataframe/column/series.py

```diff
@@ -85,6 +87,37 @@ def __getitem__(self, key: Union[int, Iterable[int]]) -> Any:
     def __len__(self) -> int:
         return self._data.__len__()
 
+    def __repr__(self) -> str:
+        indices = list(self._get_repr_indices())
+        indice_width = len(max(map(str, indices)))
+
```

Comment:
do we need to add schema like the example in docstring

Reply:
We did not integrate schema into the Dataframe and Series now..
Maybe remove schema in example docstring.